cmd/platform/allocator/command.go

```diff
@@ -26,7 +26,7 @@ import (
 // Command represents the allocator command
 var Command = &cobra.Command{
 	Use:     "allocator",
-	Short:   "Manages allocators (for ECE installations only)",
+	Short:   "Manages allocators (Requires platform administration privileges)",
```

Comment:
Maybe we can define the text as a constant and use `fmt.Sprintf()` to format this.